content/en/blog/_posts/2024-05-13-gateway-api-v1.1/index.md

```diff
@@ -0,0 +1,269 @@
+---
+layout: blog
+title: "Gateway API v1.1: Service mesh, GRPCRoute, and a whole lot more"
+date: 2024-05-13T19:00:00-08:00
+slug: gateway-api-v1-1
+author: >
+  Richard Belleville (Google),
+  Frank Budinsky (IBM),
+  Arko Dasgupta (Tetrate),
+  Flynn (Buoyant),
+  Candace Holman (Red Hat),
+  John Howard (Solo.io),
+  Christine Kim (Isovalent),
+  Mattia Lavacca (Kong),
+  Grant Spence (Red Hat),
+  Gina Yeh (Google),
+  and other reviewer and release note contributors.
+---
+
+![Gateway API Logo](gateway-api-logo.svg)
+
+Following the GA release of Gateway API last October, Kubernetes
+SIG Network is pleased to announce the v1.1 release of
+[Gateway API](https://gateway-api.sigs.k8s.io/). In this release, several features are graduating to
+_Standard Channel_ (GA), notably including support for service mesh and
+GRPCRoute. We're also introducing some new experimental features, including
+session persistence and client certificate verification. Read on for the full
+list of what's included in this release of Gateway API.
+
+**Service Mesh Support**, **GRPCRoute**, **ParentReference Port**,
+and **Conformance Profiles and Reports** will graduate to Standard.
+In addition, **Gateway Client Certificate Verification** and **Session
+Persistence and BackendLBPolicy** are adopted as new, Experimental features.
+Clarifications to TLS terminology is added to strengthen the release.
```

Comment:
Recommend leaving this level of detail out of the intro in favor of the higher level intro paragraph above.